pkgs/misc/lilypond/default.nix

```diff
@@ -75,6 +75,9 @@ stdenv.mkDerivation rec {
     "--with-texgyre-dir=${gyre-fonts}/share/fonts/truetype/"
   ];
 
+  # to ensure FlexLexer.h is found
+  NIX_CFLAGS_COMPILE = "-I${flex}/include";
```

Comment:
Wouldn't adding flex to buildInputs give the same result?

Reply:
Indeed it does.